plugins.json

```diff
@@ -95,6 +95,14 @@
     "sourceCode": "https://github.com/higlass/higlass-sequence",
     "npmPackageName": "higlass-sequence"
   },
+  {
+    "url": "https://github.com/kundajelab/higlass-dynseq#readme",
+    "title": "DynSeq Track",
+    "description": "Display a dynamic sequence in HiGlass.",
```

Comment:
How about a bit longer description like "A sequence logo track that controls the sequence height by another bigwig track" or something like that